submissions/ses3332002/memory-game/script.js

```diff
@@ -0,0 +1,164 @@
+"use strict";
+window.onload = function() {
+  
+  const themes = [
```

Comment:
You can create CSS rules and add them to the elements.

Reply:
I remade themes with CSS rules.